### PR TITLE
Configure GitHub Action to publish from GitHub to Terra.

### PR DESCRIPTION
**Pull request by deflaux**
This is configured to deploy to Terra when a release is tagged or when someone manually triggers the GitHub Action.

You can see a successfully run of https://github.com/deflaux/metis/actions/runs/368745275 has published to a test workspace https://app.terra.bio/#workspaces/verily-metis/Metis-toolkit-for-vaccine-trial-planning-test-copy

### CODE REVIEW

**Comment by deflaux** [comment]
I used the GitHub UI to create a ["release"](https://github.com/deflaux/metis/releases/tag/deploy-to-terra-3) which triggered this action: https://github.com/deflaux/metis/actions/runs/368798464

See the value of the tag reflected in the pip install command: https://app.terra.bio/#workspaces/verily-metis/Metis-toolkit-for-vaccine-trial-planning-test-copy/notebooks/launch/run_me_first.ipynb

File: .github/workflows/publish-to-terra.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    # Allows manually triggering of workflow on a selected branch via the GitHub Actions tab.
+    # GitHub blog demo: https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/.
+
+  release:
+    types: [published]
+
+env:
+  # TO REUSE THIS TEMPLATE, change these values to reflect those of your destination Terra workspace.
+  NAMESPACE: 'verily-metis'
+  # TODO(deflaux) update these after testing is complete.
```
Comment (kleined) [inline .github/workflows/publish-to-terra.yml:19]: Update?

**Review by kleined — APPROVED** [review]
This is so cool!

File: .github/workflows/publish-to-terra.yml (hunk 1 above)
Comment (deflaux) [reply .github/workflows/publish-to-terra.yml]: done!

**Pull request merged**